guides/common/modules/proc_managing-packages-on-the-base-operating-system.adoc

```diff
@@ -29,27 +29,25 @@ The `{foreman-maintain} packages` command restarts some services on the operatin
 # {package-update-project}
 ----
 
-.Using yum to Check for Package Updates
-If you want to check for updates using `yum`, enter the command to install and update packages manually and then you can use `yum` to check for updates:
+.Enabling yum for {ProjectServer} or {SmartProxyServerTitle} Package Management
+If you want to install and update packages on your system using `yum` directly and control the stability of the system yourself, enter the following command:
 
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-maintain} packages unlock
-# yum check update
-# {foreman-maintain} packages lock
```

Comment:
This was intentionally specified as a single command. We want things to be locked again. If you split it in separate chapters there's a big chance people forget it.

Reply:
Makes sense. Well, I had other options but it seems the current one fits better of all.
Is it good to close this PR?

Reply:
Also, we can remove `Enabling yum for Satellite Server or Capsule Server Package Management` and `Restoring Package Management to the Default Settings` and reword the `Using yum to Check for Package Updates`